ftw/recipe/solr/__init__.py

```diff
@@ -150,6 +150,14 @@ def install(self):
             log_dir=log_dir,
             log4j2_xml=log4j2_xml,
         ))
+        # Create configoverlay.json
+        configoverlay = self.options.get('configoverlay')
+        configoverlay_filename = os.path.join(home_dir, "configoverlay.json")
```

Comment:
This line could be moved into the if block as it need to be executed only if a `configoverlay` is defined.